### PR TITLE
Test/co 976 docker integration test

### PR DESCRIPTION
**Pull request by gmtcrary**
~~Adds a testing script for the osgeo/proj image.~~
~~Needs a Jenkin's role for running this test.~~

Changes the docker image from bad "osgeo/proj.4" to "osgeo/proj".

Update for this PR: Testing for this change is in Jenkins.

### CODE REVIEW

File: osgeoproj_test.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+export URL='https://services1.arcgis.com/dePSdaG71BplHMCO/arcgis/rest/services/TaPS_Prkg_Zones/FeatureServer/0/query/?where=1%3D1&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&outFields=*&returnGeometry=true&returnCentroid=true&multipatchOption=xyFootprint&maxAllowableOffset=&geometryPrecision=&outSR=&datumTransformation=&returnIdsOnly=false&returnCountOnly=false&returnExtentOnly=false&returnDistinctValues=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&resultOffset=&resultRecordCount=&returnZ=false&returnM=false&returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=pgeojson&token='
```
Comment (wilsonia) [inline osgeoproj_test.sh]: Not that this API is going anywhere, but would it make more sense to store this data locally? Just so this test does not have any dependency on the ArcGIS API.

**Review by wilsonia — CHANGES_REQUESTED** [review]
The test script works as expected

File: osgeoproj_test.sh (hunk 1 above)
Comment (gmtcrary) [reply osgeoproj_test.sh]: Hmm thats a good idea but considering we only use this with ArcGIS for now we can leave it as is.

I'm going to be refactoring this script into the Jenkins/Ansible role as well.

**Comment by gmtcrary** [comment]
Update: Testing has been moved to a Jenkins task.

**Review by jaredkosanovic — APPROVED** [review]

**Review by jaredkosanovic — APPROVED** [review]

**Pull request merged**